tools/configen/README.md

```diff
@@ -11,68 +11,107 @@ Configen requires config file per project. The config indicates what classes to
 the generated code etc.
 Configen will suggest creating a config on the first run:
 ```
-$ configen 
-[2020-08-14 12:06:11,359][configen.configen][ERROR] - Use --config-dir DIR.
-If you have no config dir yet use the following command to create an initial config in the `conf` dir:
-        configen init_config_dir=conf
-$ configen init_config_dir=conf
-[2020-08-14 12:07:07,752][configen.configen][INFO] - Initializing config in 'conf'
+$ configen
+[2020-08-14 12:06:11,359][configen.configen][ERROR] -
+Use --config-dir DIR --config-name NAME
+e.g:
+	configen --config-dir conf --config-name configen
+
+If you have no config dir yet use init_config_dir=DIR to create an initial config dir.
+e.g:
+	configen init_config_dir=conf
 ```
 
 This will generate a basic config similar to:
 ```yaml
-$ cat conf/configen.yaml 
+$ cat conf/configen.yaml
+defaults:
+  - configen_schema
+
 configen:
   # output directory
-  output_dir: ${hydra:runtime.cwd}/gen
+  output_dir: ${hydra:runtime.cwd}/example
 
   header: |
+    # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
     # Generated by configen, do not edit.
     # See https://github.com/facebookresearch/hydra/tree/master/tools/configen
+    # fmt: off
+    # isort:skip_file
+    # flake8: noqa
 
-  # The directory of each module under output_dir
-  module_dir: '{{module_path}}/conf'
+  module_path_pattern: 'config/{{module_path}}.py'
 
   # list of modules to generate configs for
   modules:
-    - name: configen.samples.user
+    - name: configen.samples.my_module
+      # Optional dict defining hydra flags for all classes in this module:
+      # default_flags:
+      #   _convert_ : ALL
+      #   _recursive_ : True
+      #
```

Comment:
This can be confusing without the context [here](https://hydra.cc/docs/next/patterns/instantiate_objects/overview/#disable-recursive-instantiation)

If we want to mention this could we  have a seperate section with an example? If not could we remove this from here?

Reply:
@jieru-hu I removed the comment. Thanks for the review!